flags.css

```diff
@@ -1,8 +1,56 @@
 /* Indonesia */
+.indonesiaFlag {
+    border: 2px solid;
+    height: 200px;
+    width: 300px;
+}
+.indonesiaStrip1 {
+    background-color: red;
+    height: 100px;
+}
 
 /* Colombia */
-
+.columbiaFlag {
+    margin-top: 50px;
+    border: 2px solid;
+    height: 25vh;
```

Comment:
in most cases you don't want to mix units like this

Reply:
vh and pixel

Reply:
in fact, in most cases, you don't want to use vh/vw inside another container that is itself defined by vh/vw